cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -63,6 +63,7 @@ public abstract class AbstractAssemblyOperator<C extends KubernetesClient, T ext
     protected final List<LocalObjectReference> imagePullSecrets;
     protected final KafkaVersion.Lookup versions;
     protected long operationTimeoutMs;
+    protected final boolean isPodDisruptionBudgetGeneration;
```

Comment:
Is it worth having it here? `KafkaMirrorMakerAssembly` will be deleted soon. So there will be just two classes inheriting from this - one using this field and one not using it. Maybe it is better to set this directly in KafkaBridgeAssemblyOperator and KAfkaMirrorMakerAssemblyOperator (unless we decide to ignore it completely given it has only few months to live)?

Reply:
Good point, I've removed it from `KafkaMirrorMakerAssembly` completely